cegs_portal/search/templates/search/v1/dna_feature.html

```diff
@@ -713,7 +714,7 @@ <h1>
             {% for feature, sources, targets, reo in feature_reos %}
             {% if feature.children.exists %}
                         {% for child in feature.children.all %}
-                        <tr>
+                        <tr data-href="{% url 'search:dna_features' 'ensembl' child.ensembl_id %}">
```

Comment:
does this need the javascript to set the click handler?

Reply:
Yes